content/docs/06.enterprise/04.scalability/apps.md

```diff
@@ -139,6 +156,34 @@ You can add custom tags to organize and filter apps in the App Catalog. For exam
 
 ---
 
+## App Thumbnails
+
+Design Apps with thumbnails to clearly display there intended use case or function to catalog users. To add a thumbnail to your app, upload an image file as a [namespace file](../../05.concepts/02.namespace-files.md) to the same namespace as the App's connected flow. For example, add an `.svg` (it can also be `.jpg`, `.png`, or other image file extension) to the `company.team` namespace. The example below adds `kestra-icon.svg`.
```

Comment:
```suggestion
Design Apps with thumbnails to clearly display their intended use case or function to catalog users. To add a thumbnail to your app, upload an image file as a [namespace file](../../05.concepts/02.namespace-files.md) to the same namespace as the App's connected flow. For example, add an `.svg` (it can also be `.jpg`, `.png`, or other image file extension) to the `company.team` namespace. The example below adds `kestra-icon.svg`.
```